src/psset.c

```diff
@@ -65,6 +65,47 @@ psset_assert_ps_consistent(edata_t *ps) {
 	    edata_size_get(ps) >> LG_PAGE) == edata_longest_free_range_get(ps));
 }
 
+void
+psset_insert(psset_t *psset, edata_t *ps) {
+	psset_assert_ps_consistent(ps);
+	size_t longest_free_range = edata_longest_free_range_get(ps);
+
+	if (longest_free_range == 0) {
+		psset_bin_stats_adjust(&psset->full_slab_stats, ps,
+		    /* inc */ true);
+		return;
```

Comment:
Let's add some comments regarding that the full slabs don't need to be tracked (maybe similar to in `struct psset_stats_s`).

Reply:
Will do.